lesson2/random-massive.py

```diff
@@ -0,0 +1,14 @@
+import random
+lst1 = list()
```

Comment:
Avoid global variables. Initialize this variable inside the function.
```suggestion
list1 = list()
```